presto-main/src/main/java/com/facebook/presto/operator/OperatorStats.java

```diff
@@ -70,12 +77,15 @@ public OperatorStats(
             @JsonProperty("planNodeId") PlanNodeId planNodeId,
             @JsonProperty("operatorType") String operatorType,
 
+            @JsonProperty("totalDrivers") long totalDrivers,
+
             @JsonProperty("addInputCalls") long addInputCalls,
             @JsonProperty("addInputWall") Duration addInputWall,
             @JsonProperty("addInputCpu") Duration addInputCpu,
             @JsonProperty("addInputUser") Duration addInputUser,
             @JsonProperty("inputDataSize") DataSize inputDataSize,
             @JsonProperty("inputPositions") long inputPositions,
+            @JsonProperty("inputPositionsSquared") double inputPositionsSquared,
```

Comment:
This is not `inputPositions^2`. This is sum of input positions squares from each driver:
`inputPositions_driver1^2 + inputPositions_driver2^2 + inputPositions_driver_3^2 + ...`.
This stats is required to compute online standard deviation. Do you think field name should be different?